src/sagemaker/jumpstart/model.py

```diff
@@ -786,6 +791,70 @@ def register_deploy_wrapper(*args, **kwargs):
 
         return model_package
 
+    def display_benchmark_metrics(self):
+        """Display Benchmark Metrics for deployment configs."""
+        config_names = []
+        latencies = []
+        through_puts = []
+        instance_types = []
+        instance_rates = []
+        for deployment_config in self.list_deployment_configs():
+            config_names.append(deployment_config.get("ConfigName"))
+            instance_types.append(deployment_config.get("BenchmarkMetrics").key()[0])
+
+            for benchmark_metric in deployment_config.get("BenchmarkMetrics").values():
```

Comment:
If any of the fields are potentially missing in the deployment configuration, there is a chance we lose order in lists

DC = { {l=A, t=A1, c=A2}, {l=B t=B1, c=}, {l=C, t=C1, c=C2}}


[A, B, C] -> latency
[A1, B1, C1] -> throughput
[A2, C2] -> cost